sycl/include/CL/sycl/accessor.hpp

```diff
@@ -669,9 +678,18 @@ class accessor :
   using AccessorSubscript =
       typename AccessorCommonT::template AccessorSubscript<Dims>;
 
-  using RefType = DataT &;
   using ConcreteASPtrType = typename detail::PtrValueType<DataT, AS>::type *;
+#if defined(RESTRICT_WRITE_ACCESS_TO_CONSTANT_PTR)
+  using RefType = typename std::conditional<
+      AS == access::address_space::constant_space,
+      const DataT &, DataT &>::type;
```

Comment:
It's kinda funny that clang-format chose to treat identical blocks of code differently. Could this be aligned with the other `RefType` declarations?

Reply:
Done.